### PR TITLE
GOLDEN BLAST: a new judo combo

### PR DESCRIPTION
**Pull request by silverplatedelta**
## What Does This PR Do
Adds a new combo that requires 15 inputs, upon completion will stun (300 stamina damage), knock down, and confuse the victim.

The Combo:
Help, Disarm, Help, Grab, Disarm, Disarm, Grab, Help, Disarm, Disarm, Grab, Help

## Why It's Good For The Game
It was suggested on the forms and fairly well received. I also chuckled at the suggestion. given the 15 inputs (compared to the 3 of a baton) to stun, balance implications are quite low. 

Its also very silly. Have fun getting yelled at for help intent swapping in combat or helping your opponent up. 

## Testing
Code complies, combo executes properly, no runtimes were thrown.

## Changelog
:cl:
add: Adds a new combo to Corp Judo: Golden Blast
/:cl:

### CODE REVIEW

**Comment by Sayrbee** [comment]
what kind of fucking street fighter combo bullshit is this I love it

**Comment by lewcc** [comment]
Finally Tekken 13

File: code/modules/martial_arts/combos/judo/goldenblast.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/datum/martial_combo/judo/goldenblast
+	//harm disarm harm grab swap disarm disarm grab swap harm disarm disarm grab swap harm
+	//this is inredibly stupid. I love it.
+	name = "Golden Blast"
+	steps = list(MARTIAL_COMBO_STEP_HELP, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_HELP, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HELP, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HELP)
+	explaination_text = "Through use of a martial arts form, you can stun a foe with life energy. Or by overcharging the belt's nanites. Take your Pick."
+	combo_text_override = "Help, Disarm, Help, Grab, Disarm, Disarm, Grab, Help, Disarm, Disarm, Grab, Help"
+
+/datum/martial_combo/judo/goldenblast/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	target.visible_message("<span class='warning'>[user] blasts [target] with and energy blast, and sends them to the ground!</span>", \
```
Comment (lewcc) [inline code/modules/martial_arts/combos/judo/goldenblast.dm:10]:
```suggestion
	target.visible_message("<span class='warning'>[user] blasts [target] with an energy blast, sending [target.p_them()] to the ground!</span>", \
```

File: code/modules/martial_arts/combos/judo/goldenblast.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/datum/martial_combo/judo/goldenblast
+	//harm disarm harm grab swap disarm disarm grab swap harm disarm disarm grab swap harm
```
Comment (lewcc) [inline code/modules/martial_arts/combos/judo/goldenblast.dm:2]: Comment doesn't match the actual combo?

Comment (silverplatedelta) [reply code/modules/martial_arts/combos/judo/goldenblast.dm:2]: forgot to remove that! that was for me when I was porting it from my notes to code.

**Comment by silverplatedelta** [comment]
latest commit reduces stam damage to 120 (as that is the max and I was unaware). No mechanical changes come as a result of this change.

**Comment by Verbalase** [comment]
space fighter 13!???

meh tekken 13 moreish

**Comment by spacegeese** [comment]
can you add flavor text for the attacks, to show how absolutely fucked the victim on the receiving end of this is? for example:

"Officer Beatsky shakes your hand and glares directly into your eyes! You feel like you're supremely fucked!"
"Sergeant Sturdy grabs you by the ear and waggles it around!"
"John Robust makes strange hand gestures, screams wildly and pokes you directly in the chest! You feel the wrath of the **_GOLDEN BOLT_** surge through your body! You've been utterly robusted!"

**Comment by Mitchs98** [comment]
I love this and will absolutely main Judo for a week if it's merged.

**Comment by MisterMan13** [comment]
Made a github account to say how much I love that this is an actual thing now. Silverplate I'm your #1 fan

**Review by Burzah — APPROVED** [review]

File: code/modules/martial_arts/combos/judo/goldenblast.dm [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/datum/martial_combo/judo/goldenblast
+	//this is inredibly stupid. I love it.
```
Comment (DGamerL) [inline code/modules/martial_arts/combos/judo/goldenblast.dm:2]:
```suggestion
	// This is incredibly stupid. I love it.
```

**Review by DGamerL — APPROVED** [review]

**Comment by ErdGinalD** [comment]
someone tell him there is 120 maxcap of stamina damage

**Pull request merged**